### PR TITLE
GSREN3D-123: add Trambus layer

### PR DESCRIPTION
**Pull request by fgravin**
### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-123

### Description

Supersed https://github.com/camptocamp/rennes3d-front/pull/23

- add terrain and building layers (not working yet)
- add WFS as geojsonLayer for TramBus and POIS
- add temp POIS data

### CODE REVIEW

File: src/map.config.json [hunk 1]
```diff
@@ -58,7 +70,7 @@
       "format": "image/jpeg",
       "tileMatrixSetID": "EPSG:3857",
       "tileMatrixPrefix": "EPSG:3857:",
-      "activeOnStartup": false
+      "activeOnStartup": true
```
Comment (ismailsunni) [inline src/map.config.json:73]: CMIIW, this Rennes ortho is used for 3D map. Is it correct to activate it on startup (I thought the default is 2D map)

File: src/map.config.json [hunk 2]
```diff
@@ -117,20 +164,28 @@
...
-      "activeOnStartup": true,
-      "layers": "trp_coll:trambus_lignes",
-      "parameters": "format=image/png&transparent=true"
-    },
-    {
-      "name": "trambusStops",
-      "url": "https://public.sig.rennesmetropole.fr/geoserver/ows",
-      "type": "WMSLayer",
+      "name": "poi",
+      "url": "/src/assets/data/POI.json",
```
Comment (ismailsunni) [inline src/map.config.json:168]: Since it is data, should we put it outside the repo?

File: src/map.config.json (hunk 1 above)
Comment (plaulhe) [reply src/map.config.json:73]: i agree with Ismail.

File: src/map.config.json (hunk 2 above)
Comment (fgravin) [reply src/map.config.json:168]: yes we can do that

File: src/map.config.json (hunk 1 above)
Comment (fgravin) [reply src/map.config.json:73]: It's just more readable for now.
Anyway, the config and the styles are clearly work in progress, it's not a big deal ;)

Comment (plaulhe) [reply src/map.config.json:73]: Not at all.

**Pull request merged**